code/modules/mob/mob.dm

```diff
@@ -494,7 +494,7 @@ var/global/obj/screen/fuckstat/FUCK = new
 		else if(hand_index)
 			put_in_hand(hand_index, W)
 
-	if(ishuman(src) && W == src:head)
+	if(ishuman(src) && W == src:head) //AAAAAUGH
```

Comment:
God the least you could do is make this use a cast BECAUSE THIS CODE WILL BREAK IF INVENTORY EVER GETS REFACTORED AND IT WON'T COMPILE ERROR.